catroid/build.gradle

```diff
@@ -151,7 +151,7 @@ dependencies {
     compile fileTree(include: '*.jar', dir: 'src/main/libs-natives')
     androidTestCompile fileTree(include: '*.jar', dir: 'src/androidTest/libs')
     androidTestCompile 'com.jayway.android.robotium:robotium-solo:5.2.1'
-    androidTestCompile 'com.google.dexmaker:dexmaker-mockito:1.2'
+    androidTestCompile 'com.linkedin.dexmaker:dexmaker-mockito:1.5.1'
```

Comment:
It is necessary, if you want to use the 'spy'-feature of mockito which I did in my test. 1.2 was the highest available version by google, so this does seem like the only way to up the version.

However if we don't want this I will need to find another way to write a test.